Help/FAQ.rst

```diff
@@ -55,6 +55,14 @@ There are also generic bears, which can be applied language independently on
 your code. Their capabilities and information can be seen
 `here <https://coala.io/#/languages>`__.
 
+What is a bear anyway?
+----------------------
+
+A bear is a routine that is used by a coala package to check for lints and
+fix issues if found. A group of these routines is used for defining certain
+quality requirements in various languages and these checking routines are
```

Comment:
I think you should change this and make it more straightforward to understand. You need not define the internals of bears, a bear itself should be considered as a rule set for analysis instead of its routines.

Also, maybe mention that bears are user configurable.

Reply:
@ksdme I have confirmed this definition of bear from @jayvdb . And he approved that.

Reply:
I would still recommend lints => lint issues and optionally fix if automatic fixes are available.

Reply:
Just noting that my approval was only that what you had written was good.  But we have a team of developers who can help fine tune the wording for you ;-)